web/app/view/GroupDialog.js

```diff
@@ -40,5 +40,18 @@ Ext.define('Traccar.view.GroupDialog', {
             displayField: 'name',
             valueField: 'id'
         }]
-    }
+    },
+
+    buttons: [{
```

Comment:
It was my first decision, but was something wrong and page stopped loading.
Geofences also have attributes, but we do not use it for now. I'll try how it will look with 4 buttons ( + Area).